connectors/webhook/element-templates/webhook-connector-intermediate.json

```diff
@@ -229,7 +229,7 @@
       ],
       "binding": {
         "type": "zeebe:property",
-        "name": "inbound.authorizationType"
+        "name": "inbound.auth.type"
```

Comment:
Since we're changing model, I think we need to bump an element template version.